src/sagemaker/jumpstart/filters.py

```diff
@@ -62,9 +96,32 @@ def eval(self) -> None:
         """Evaluates operand."""
         return
 
+    @property
+    def resolved_value(self):
+        """Getter method for resolved_value."""
+        return self._resolved_value
+
+    @resolved_value.setter
+    def resolved_value(self, new_resolved_value: Any):
+        """Setter method for resolved_value. Resolved_value must be of type ``BooleanValues``."""
+        if isinstance(new_resolved_value, BooleanValues):
+            self._resolved_value = new_resolved_value
+            return
+        raise RuntimeError(
+            "Resolved value must be of type BooleanValues, "
+            f"but got type {type(new_resolved_value)}."
+        )
+
     @staticmethod
     def validate_operand(operand: Any) -> Any:
```

Comment:
The problem with `@staticmethod` in Python is that you cannot return the own type, which makes it harder to write factory pattern for example. Consider changing this method to a standalone utility, such that you can write:

```py
def validate_operand(operand: Any) -> Operand:
   ...
```

Reply:
I personally think the benefit we get from having this as a class method, from an organization and extensibility standpoint, outweighs the `Any` return type.